randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -613,9 +616,21 @@ Extra - room_id: [16]
       Any of the following:
           All of the following:
               Gravity Suit
-              Space Jump or Wall Jump (Advanced)
-          # Cannot freeze Evir with missiles
-          Underwater Wall Jump (Expert) and Can Freeze Enemies
+              Any of the following:
+                  Space Jump
+                  # With WJ:
+                  Wall Jump (Advanced)
+                  All of the following:
+                      Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
```

Comment:
previous comment said the evir cannot be frozen with missiles. is that false?

Reply:
it is false